lib/threema/receive/delivery_receipt.rb

```diff
@@ -1,12 +1,46 @@
 # frozen_string_literal: true
 
+require 'threema/util'
+
 class Threema
   module Receive
     class DeliveryReceipt
-      attr_reader :content
+      STATUS_TYPE_BYTE = {
```

Comment:
```suggestion
      STATUS_BYTE_MAPPING = {
```

Reply:
Like this naming better, I tried to copy the verbage from the `Threema::TypedMessage` class where we also have a type mapping, but no need to copy if there is some improvement that could be made.